priv/repo/migrations/20240705175329_add_badge_serials.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Orbit.Repo.Migrations.AddBadgeSerials do
+  use Ecto.Migration
+
+  def change do
+    create table("badge_serials") do
+      add :employee_id, references("employees", on_delete: :delete_all)
```

Comment:
Realistically we've never going to actually delete from `employees`, right?

Reply:
Yeah, if we need something akin to "delete" functionality in the application we can always make a soft delete, but the table entry would stick around for the purposes of having a record of past sign-ins and stuff.